pkg/v1/tkg/tkgconfigupdater/tkgconfigupdater_test.go

```diff
@@ -584,8 +584,8 @@ var _ = Describe("EnsureProviders", func() {
 			err = copyData(tkgConfigNode, &userProviders)
 			Expect(err).ToNot(HaveOccurred())
 
-			Expect(userProviders.CertManager.URL).To(ContainSubstring("providers/cert-manager/v1.5.3/cert-manager.yaml"))
-			Expect(userProviders.CertManager.Version).To(Equal("v1.5.3"))
+			Expect(userProviders.CertManager.URL).To(ContainSubstring("providers/cert-manager/v1.7.2/cert-manager.yaml"))
+			Expect(userProviders.CertManager.Version).To(Equal("v1.7.2"))
```

Comment:
This is part of the unit tests and after looking at the unit test on how it is implemented it looks like it is using dummy TKG BoM file and runs the test with it. Meaning providers that we might be fetching with the dummy TKG BoM has `v1.5.3` defined and hence the unit tests are failing.
(It looks like this unit tests is complex we should refactor a bit in future)